src/platform/forms-system/src/js/constants.js

```diff
@@ -1,2 +0,0 @@
-export const REVIEW_APP_DEFAULT_MESSAGE = 'Review Application';
-export const APP_TYPE_DEFAULT = 'application';
```

Comment:
Please see other comment regarding constant file that was moved and renamed.

Reply:
Done.